src/Simulation/Simulators.Tests/QuantumSimulatorTests/BasicTests.cs

```diff
@@ -20,12 +20,22 @@ public void QSimConstructor()
             {
                 Assert.Equal("Quantum Simulator", subject.Name);
             }
+            using (var subject = new SparseSimulator2())
+            {
+                Assert.Equal("SparseSimulator2", subject.Name);
+            }
```

Comment:
This may need to be a separate unit test. I'd also suggest using the declaration form of `using`:

```suggestion
            using var subject = new SparseSimulator2();
            Assert.Equal("SparseSimulator2", subject.Name);
```